### PR TITLE
add SchemaHero as a 3rd-party tool

### PR DESCRIPTION
**Pull request by laverya**
SchemaHero is a Kubernetes-native implementation of declarative database schema management.

### CODE REVIEW

File: v21.1/community-tooling.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ If you have a tested or developed a third-party tool with CockroachDB, and would
 - [php-pgsql](build-a-php-app-with-cockroachdb.html)
 - [RPostgres](https://blog.ervits.com/2020/02/exploring-cockroachdb-with-r-and.html)
 - [Rust-Postgres](build-a-rust-app-with-cockroachdb.html)
+- [SchemaHero](https://schemahero.io/docs/installing/kubectl/)
```
Comment (vy-ton) [inline v21.1/community-tooling.md:23]: @laverya Can we link to the page mentioning CockroachDB support, https://schemahero.io/databases/cockroachdb/connecting/?

File: v21.1/community-tooling.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ If you have a tested or developed a third-party tool with CockroachDB, and would
...
 
 ## Visualization tools
```
Comment (vy-ton) [inline v21.1/community-tooling.md:25]: Let's create a header for `Schema migration tools`

cc @ericharmeling

**Review by vy-ton — APPROVED** [review]
@laverya Thank you for building/adding CockroachDB support here!

2 minor comments before merge

**Comment by vy-ton** [comment]
@ericharmeling ready for your review/merge

**Review by ericharmeling — APPROVED** [review]
LGTM

@laverya Thank you for the contribution!

**Comment by laverya** [comment]
I don't know what's up with the "list files changed" action - anything I should be doing?

**Comment by ericharmeling** [comment]
> I don't know what's up with the "list files changed" action - anything I should be doing?

You just added that one line to one file, so I think it’s just an issue with the checks. The build passed just fine. I went ahead and merged.

**Pull request merged**